app/gui/view/graph-editor/src/builtin/visualization/native/raw_text.rs

```diff
@@ -147,6 +161,15 @@ impl RawTextModel {
     }
 }
 
+fn split_long_lines(data_str: &String, process_line: &mut impl FnMut(String) -> Result<(), DataError>) -> Result<(), DataError> {
```

Comment:
```suggestion
fn split_long_lines(data_str: &str, process_line: &mut impl FnMut(String) -> Result<(), DataError>) -> Result<(), DataError> {
```

In Rust, `&String` is not usually used, but `&str` is used instead.